src/core/hyper-ref/index.js

```diff
@@ -33,6 +33,16 @@ import VisibilityDetectingView from 'hyperview/src/VisibilityDetectingView';
 import { XMLSerializer } from 'xmldom-instawork';
 import { createTestProps } from 'hyperview/src/services';
 
+export const createEventHandler = (
```

Comment:
Can you add a comment why we're using this wrapper?

Reply:
Added in [77fda83](https://github.com/Instawork/hyperview/pull/453/commits/77fda838f90af11274ea8fb25d47f5bad93738f3)